packages/scripts/tests/e2e/specs/test.js

```diff
@@ -0,0 +1,14 @@
+// For authoring Nightwatch tests, see
+// http://nightwatchjs.org/guide#usage
+
+module.exports = {
+  'default e2e tests': browser => {
+    browser
```

Comment:
please update this to basic test , would be even nice to test basic functions 
- list
- add 
- edit

Reply:
I removed the e2e test for now. I think this is something to think about and have a good talk about. I would like the test, but i do not like to have molgenis in the background to do so. Mocking a api in a e2e is not best practice.

Reply:
working e2e test added